### PR TITLE
Ghost "Blinky" Logic

### PR DESCRIPTION
**Pull request by kaushikb11**
Resolves Issue #50 
Atomic PR for the functionality of Blinky ghost behavior of following the Pacman.

### CODE REVIEW

File: client/src/components/PacmanGame/PacmanGame.jsx [hunk 1]
```diff
@@ -170,6 +171,58 @@ class PacmanGame extends Component {
     }
   }
 
+  getGridwithWeights = (grid) => {
+    const gridwithWeights = [];
+    grid.forEach((array) => {
```
Comment (Shalinit3) [inline client/src/components/PacmanGame/PacmanGame.jsx]: You can use map function here.

File: client/src/components/PacmanGame/PacmanGame.jsx [hunk 2]
```diff
@@ -170,6 +171,58 @@ class PacmanGame extends Component {
...
+      const newArray = [];
+      array.forEach((element) => {
```
Comment (Shalinit3) [inline client/src/components/PacmanGame/PacmanGame.jsx]: This forEach can also be replaced by map

**Review by Shalinit3 — CHANGES_REQUESTED** [review]

File: client/src/components/PacmanGame/PacmanGame.jsx [hunk 3]
```diff
@@ -170,6 +171,58 @@ class PacmanGame extends Component {
...
+        let weight = element;
+        if (weight === 4) {
```
Comment (Shalinit3) [inline client/src/components/PacmanGame/PacmanGame.jsx:178]: This can be done by ternary operator.

**Review by 318097 — APPROVED** [review]

**Review by Shalinit3 — APPROVED** [review]

File: client/src/components/PacmanGame/PacmanGame.jsx (hunk 3 above)
Comment (kaushikb11) [reply client/src/components/PacmanGame/PacmanGame.jsx:178]: Readability purpose.

**Pull request merged**